src/index.esm.js

```diff
@@ -4,3 +4,20 @@ export * from './elements';
 export * from './platform';
 export * from './plugins';
 export * from './scales';
+
+export * as controllers from './controllers';
```

Comment:
I think this is too new:
> export * as name1 from …; // Draft ECMAScript® 2O21

I think you can just do `export {controllers, elements, plugins, scales};` after the imports.

Reply:
I switched to that, tested locally, and everything still works with tree-shaking.

Reply:
I can reproduce the error locally. Changing `ecmaVersion: 2019` to `ecmaVersion: 2020` in eslintrc.yml removes the error too.